<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMsgBox</class>
 <widget class="QWidget" name="DlgMsgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>464</width>
       <height>276</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>464</width>
       <height>276</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FAFDFF;
border-radius: 8px;
border: none;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>56</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>56</number>
      </property>
      <property name="bottomMargin">
       <number>56</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="labelTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>96</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>96</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLabel#labelTitle{
font-size: 22px;
font-weight: 500;
color: #0047BB;
background-position: left center;
background-repeat:no-repeat;
padding-left:40px;
background-image: url(:/image/images/tipsok.png);
}
.QLabel#labelTitle[state=&quot;ok&quot;]{
color: #0047BB;
background-image: url(:/image/images/tipsok.png);
}
.QLabel#labelTitle[state=&quot;error&quot;]{
color: #CE4949;
background-image: url(:/image/images/tipserror.png);
}
.QLabel#labelTitle[state=&quot;warn&quot;]{
color: #EEC01C;
background-image: url(:/image/images/tipswarn.png);
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTitleNo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 22px;
font-weight: 500;
color: #2D3440;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16px;
font-weight: normal;
color: #5E626E;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnNo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>44</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>44</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 3px;
background: #FAFDFF;
border: 1px solid #DFE3E5;
font-size: 16px;
font-weight: normal;
color: #2D3440;</string>
          </property>
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnYes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>44</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>44</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #0047BB;
border-radius: 3px;
font-size: 16px;
font-weight: normal;
color: #FAFDFF;</string>
          </property>
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
